frontend/tailwind.config.ts

```diff
@@ -69,6 +70,7 @@ export default {
 					1: "#cdd993",
 					2: "#a7c957",
 					3: "#6a994e",
+					"-1": "#274e88",
```

Comment:
For me it's a bit hard to read the mouseover text on this blue, maybe the blue should be a bit lighter?
![image](https://github.com/user-attachments/assets/c9f485b6-57c7-405a-a807-7418c0799cd8)